src/main/kotlin/com/hexagonkt/todokt/backend/TaskRouter.kt

```diff
@@ -0,0 +1,103 @@
+package com.hexagonkt.todokt.backend
+
+import com.hexagonkt.http.server.Call
+import com.hexagonkt.http.server.Router
+import com.hexagonkt.serialization.Json
+import com.hexagonkt.todokt.backend.entities.Task
+import com.hexagonkt.todokt.backend.stores.TaskStore
+import java.util.*
+import kotlin.text.Charsets.UTF_8
+
+
+internal val router = Router {
+
+    val store = injector.inject<TaskStore>(Task::class)
+
+    cors()
+
+    options("/*") { }
```

Comment:
Is this route required? if so, it would be great to explain its purpose with a comment, as it is unusual to see an empty route.